source/clear-linux/get-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -20,7 +20,7 @@ Look for the :file:`clear-[version number]-installer.img.xz` file. You can also
 
 .. code-block:: bash
    
-   curl -O https://download.clearlinux.org/image/clear-$(curl https://download.clearlinux.org/latest)-installer.img.xz
+   curl -O https://download.clearlinux.org/image/$(curl https://download/clearlinux.org/image/latest-images | grep "installer")

```

Comment:
There is a typo in the nested curl command: `download/clearlinux.org` -> `download.clearlinux.org`.

Otherwise, this change looks good to me.

Reply:
Thanks for catching that!

Reply:
As long as those changes are made the revised curl script is good. I tested it on my NUC too.

Reply:
change made